CMakeLists.txt

```diff
@@ -278,6 +278,16 @@ elseif(UNIX)
     target_compile_definitions(PortAudio PRIVATE PA_USE_COREAUDIO=1)
     set(PKGCONFIG_LDFLAGS_PRIVATE
       "${PKGCONFIG_LDFLAGS_PRIVATE} -framework CoreAudio -framework AudioToolbox -framework AudioUnit -framework CoreFoundation -framework CoreServices")
+  elseif(ANDROID)
```

Comment:
Why is this gated on both UNIX and ANDROID? OpenSLES is not exclusive to either. I think this should be treated like JACK and allowed on all operating systems if the required library is found.

Reply:
The implementation does contain some android-specific code, which could be preprocessored out where necessary of course to make this a universal unix implementation.

Reply:
If you want to make this compatible with non-Android systems, great, but otherwise I think it's fine to add a comment here in CMakeLists.txt explaining this has Android specific code even though OpenSL ES is not exclusive to Android.

Reply:
I don't know of any systems besides Android that support OpenSL ES.